src/hyperion/external_interaction/ispyb/ispyb_store.py

```diff
@@ -53,49 +53,57 @@ def begin_deposition(
         data_collection_group_info: DataCollectionGroupInfo,
         scan_data_info: ScanDataInfo,
     ) -> IspybIds:
-        with ispyb.open(self.ISPYB_CONFIG_PATH) as conn:
-            assert conn is not None
-            assert (
-                scan_data_info.data_collection_info.visit_string
-            ), "No visit string supplied for ispyb"
-            data_collection_group_id = scan_data_info.data_collection_info.parent_id
-            if not data_collection_group_id:
-                data_collection_group_id = self._store_data_collection_group_table(
-                    conn, data_collection_group_info
-                )
-                scan_data_info.data_collection_info.parent_id = data_collection_group_id
-            data_collection_id = self._store_data_collection_table(
-                conn, None, scan_data_info.data_collection_info
-            )
-            return IspybIds(
-                data_collection_group_id=data_collection_group_id,
-                data_collection_ids=(data_collection_id,),
+        ispyb_ids = IspybIds()
+        if scan_data_info.data_collection_info:
```

Comment:
The creating/updating of groups is very complicated, I've made https://github.com/DiamondLightSource/hyperion/issues/1293 to simplify